emis/urls.py

```diff
@@ -15,8 +15,15 @@
 """
 from django.contrib import admin
 from django.urls import path, include
+from django.conf import settings
+from django.conf.urls.static import static
 
 urlpatterns = [
-    path('', include('web_app.urls')),
     path('admin/', admin.site.urls),
+    path('', include('historical_surveillance.urls', namespace='surveillance')),
```

Comment:
In general this is good practice to enforce a psuedo namespacing here. 

A simple example is If historical surveillance has a URL named "admin" it could conflict with the admin routes. 

So to avoid that is to have each app namespace the URLs

All historical app URLs start with `/historical/<something>`
All school app URLs are formatted `/school/<something`
etc

Reply:
Yes. That is the idea except there is a better way.